soc/arm/st_stm32/stm32g4/power.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics.
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
```

Comment:
This doesn't look accurate.

Reply:
fixed
this file is still based on G0 implementation so I kept the previous copyright notices and added a new one